src/Tribe/Service_Providers/Context.php

```diff
@@ -178,17 +201,22 @@ static function () {
 					 * var first and get, from its position, a clean variable we can consume in Views.
 					 */
 					'read' => [
-						Tribe__Context::REQUEST_VAR => [ 'view', 'tribe_view', 'tribe_event_display', 'eventDisplay' ],
+						Tribe__Context::REQUEST_VAR => [
+							$view_query_var,
+							'tribe_view',
+							'tribe_event_display',
+							'eventDisplay'
```

Comment:
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma
* :no_entry_sign: Comma required after last value in array declaration

<sub>hash: f3c34c866fdd9ae26745db2695820dc3</sub>